code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -112,12 +114,22 @@
 			else								//Everyone else fails, skip the emote attempt
 				return
 
+		if("chirp", "chirps")
+			if(isvox(src)) //Only Vox can chirp
+				on_CD = handle_emote_CD()			//proc located in code\modules\mob\emote.dm'
+			else								//Everyone else fails, skip the emote attempt
+				return
+
 		if("scream", "screams")
 			on_CD = handle_emote_CD(50) //longer cooldown
 		if("fart", "farts", "flip", "flips", "snap", "snaps")
 			on_CD = handle_emote_CD()				//proc located in code\modules\mob\emote.dm
 		if("cough", "coughs", "slap", "slaps", "highfive")
 			on_CD = handle_emote_CD()
+		if("gasp", "gasps")
```

Comment:
Not sure why this is shown as an added line of code... might be due to the outdated branch